src/components/LiveGraph.tsx

```diff
@@ -81,15 +81,19 @@ const LiveGraph = () => {
   const [isRecording, setIsRecording] = useState<boolean>(false);
   useEffect(() => {
```

Comment:
This was just faffing/tidying, but in a future PR I'd like to be able to use the actual recording times here... not a similar time after a re-render.